webproject/src/main/webapp/js/chat-box.js

```diff
@@ -23,6 +27,12 @@ $(document).ready(function() {
     });
 });
 
+function getUsername() {
```

Comment:
I feel that the correct way to do this is to initialize username with null, and call getUsername() in sendMessage() if the username variable is null
this way we can ensure that the username variable is not null before the message was sent.

however, the right way to do this is still to send it through backend - which we will come back later.
as of now, people are able to inspect the page and write code like
```
username = "barrack obama"
```
and it will replace their name as "barrack obama" which is wrong

Reply:
Alright! The backend is already in master though